### DISCUSSION TITLE
How fast is this in terms of user authentifications per second?

### DISCUSSION THREAD

**Discussion by tronics** [nextauthjs/next-auth#5617] [Help]
Hello,

I am looking to find an authsystem that allows for 1k to 10k user authentifications per second and partition on multiple servers for more. Is this the right place to look here? What are your experiences, and how do you tweak it?
Thank you.

**Comment by MJomaa** [comment]
10k authentications per second equal to 864 million per day. What do you consider as authentication and what is the use case here?

**Reply by tronics** [reply]
10k is peak only. I have a drop for sneakers that should receive logins from many of the 500k already preregistered users within the first 5 minutes.
This performance of 1k-10k is needed is for server side authentification only: password check, db lookup, session.
I'd be fine with additional infrastructure but I would like to know how well next-auth can perform.
